src/ontology/mondo-edit.obo

```diff
@@ -274763,11 +274763,11 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0013772
-name: congenital cataract-hearing loss-severe developmental delay syndrome
+name: Huppke-Brendel syndrome
 subset: ordo_disease {source="Orphanet:300313"}
 synonym: "CCHLND" RELATED ABBREVIATION [MONDO:Lexical, OMIM:614482]
-synonym: "congenital cataract-deafness-severe developmental delay syndrome" EXACT [Orphanet:300313]
```

Comment:
why was this synonym referencing Orphanet removed?

Reply:
It looks like a mistake. I think I may have meant to combine the two syns below and accidentally deleted this instead?

I'll fix

Reply:
thank you!